docs/core/preview3/tutorials/using-with-xplat-cli-msbuild.md

```diff
@@ -1,190 +1,107 @@
 
 2. `$ dotnet restore`
 
-   [`dotnet restore`](../tools/dotnet-restore.md) calls into [NuGet](http://nuget.org) (.NET's package manager) to restore the tree of dependencies. NuGet analyzes the `Hello.csproj` file, downloads the dependencies stated in the file (or grabs them from a cache on your machine), and writes the `obj/project.assets.json` file.  The `project.assets.json` file is necessary to be able to compile and run.
+   [`dotnet restore`](../tools/dotnet-restore.md) calls into [NuGet](http://nuget.org) (.NET package manager) to restore the tree of dependencies. NuGet analyzes the *Hello.csproj* file, downloads the dependencies stated in the file (or grabs them from a cache on your machine), and writes the *obj/project.assets.json* file.  The *project.assets.json* file is necessary to be able to compile and run.
    
-   The `project.assets.json` file is a persisted and complete set of the graph of NuGet dependencies and other information describing an app.  This file is read by other tools, such as `dotnet build` and `dotnet run`, enabling them to process the source code with a correct set of NuGet dependencies and binding resolutions.
+   The *project.assets.json* file is a persisted and complete set of the graph of NuGet dependencies and other information describing an app.  This file is read by other tools, such as [`dotnet build`](../tools/dotnet-build.md) and [`dotnet run`](../tools/dotnet-run.md), enabling them to process the source code with a correct set of NuGet dependencies and binding resolutions.
    
 3. `$ dotnet run`
 
-   [`dotnet run`](../tools/dotnet-run.md) calls `dotnet build` to ensure that the build targets have been built, and then calls `dotnet <assembly.dll>` to run the target application.
+   [`dotnet run`](../tools/dotnet-run.md) calls [`dotnet build`](../tools/dotnet-build.md) to ensure that the build targets have been built, and then calls `dotnet <assembly.dll>` to run the target application.
    
     ```
     $ dotnet run
     Hello World!
     ```
 
-    Alternatively, you can also execute [`dotnet build`](../tools/dotnet-build.md) to compile the code without running the build console applications. This results in a `bin/Debug/netcoreapp1.0/Hello.dll` compiled application that can be run with `dotnet bin\Debug\netcoreapp1.0\Hello.dll` on Windows, and `dotnet bin/Debug/netcoreapp1.0/Hello.dll` on other systems. You may specify an additional parameter on the command-line (assuming you are on Windows):
+    Alternatively, you can also execute [`dotnet build`](../tools/dotnet-build.md) to compile the code without running the build console applications. This results in a *bin/Debug/netcoreapp1.0/Hello.dll* compiled application that can be run with `dotnet bin\Debug\netcoreapp1.0\Hello.dll` on Windows, and `dotnet bin/Debug/netcoreapp1.0/Hello.dll` on other systems. You may specify an additional parameter on the command-line as you'll see later on the topic.
```

Comment:
this paragraph seems to be inaccurate since the running is the same for both Windows and non-Windows and I didn't know how to change. @blackdwarf can you help?